src/assets/wise5/authoringTool/edit-branch/edit-branch.component.ts

```diff
@@ -0,0 +1,260 @@
+import { ChangeDetectorRef, Component } from '@angular/core';
+import { SelectBranchCriteriaComponent } from '../select-branch-criteria/select-branch-criteria.component';
+import { copy } from '../../common/object/object';
+import { ActivatedRoute, Router, RouterModule } from '@angular/router';
+import { MatButtonModule } from '@angular/material/button';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { SelectComponentComponent } from '../../../../app/authoring-tool/select-component/select-component.component';
+import { SelectStepComponent } from '../../../../app/authoring-tool/select-step/select-step.component';
+import { AbstractBranchAuthoringComponent } from '../abstract-branch-authoring/abstract-branch-authoring.component';
+import { BranchPathAuthoringComponent } from '../branch-path-authoring/branch-path-authoring.component';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
+import { TeacherProjectService } from '../../services/teacherProjectService';
+
+@Component({
+  imports: [
+    BranchPathAuthoringComponent,
+    FlexLayoutModule,
+    MatButtonModule,
+    ReactiveFormsModule,
+    RouterModule,
+    SelectBranchCriteriaComponent,
+    SelectComponentComponent,
+    SelectStepComponent
+  ],
+  selector: 'edit-branch',
+  standalone: true,
+  styleUrl: './edit-branch.component.scss',
+  templateUrl: './edit-branch.component.html'
+})
+export class EditBranchComponent extends AbstractBranchAuthoringComponent {
+  private branchPaths: any[] = [];
+  protected mergeStepTitle: string;
+  private node: any;
+
+  constructor(
+    private changeDetector: ChangeDetectorRef,
+    protected fb: FormBuilder,
+    protected projectService: TeacherProjectService,
+    protected route: ActivatedRoute,
+    protected router: Router
+  ) {
+    super(fb, projectService, route, router);
+  }
+
+  ngOnInit(): void {
+    super.ngOnInit();
+    const branch = this.projectService.getBranchesByBranchStartPointNodeId(this.targetId)[0];
+    this.setPathCount(branch.paths.length);
+    const mergeStepId = branch.endPoint;
+    this.setMergeStep(mergeStepId);
+    this.mergeStepTitle = this.projectService.getNodePositionAndTitle(mergeStepId);
```

Comment:
Do we need the mergeStepId variable?
```suggestion
    this.setMergeStep(branch.endPoint);
    this.mergeStepTitle = this.projectService.getNodePositionAndTitle(branch.endPoint);
```